docs/pages/blog/nextgen-pickers.md

```diff
@@ -0,0 +1,137 @@
+</video>
+
+As you can notice, the day and year leak to the previous sections of the date, presenting a serious challenge to usability.
+
+In contrast, the new fields are specialized in editing date and time values.
+They recognize the values on the input (day, month, year, etc.) and respond accordingly.
+
+<video style="max-width:608px " autoplay muted loop playsinline >
+  <source src="/static/blog/nextgen-pickers/basic-date-field.mp4" type="video/mp4" />
+</video>
+
+The Fields are included in the pickers, but they're available as standalone components too.
+You can import them from the latest v6 package.
+
+```jsx
+import { DateField } from '@mui/x-date-pickers/DateField';
+
+<DateField label="My first field" />;
+```
+
+They're also included by default with the new Pickers.
+
+```jsx
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+
+<DatePicker label="My first v6 picker" />;
+```
+
+## Enhanced keyboard usability
+
+With the new Fields, the keyboard interaction is a first-class experience. Not only can you type the date as text, but you can also use your arrow keys to navigate and edit date and time values.
+
+<video style="margin-bottom:5px; max-width:608px" autoplay muted loop playsinline >
+  <source src="/static/blog/nextgen-pickers/date-field-navigation.mp4" type="video/mp4" />
+</video>
+
+Moreover, notice on the following example that as we increase the month, the component automatically watches for valid values throughout each piece of your date input, like you're navigating in a calendar.
+
+<video style="margin-bottom:5px; max-width:608px" autoplay muted loop playsinline >
+  <source src="/static/blog/nextgen-pickers/smart-field.mp4" type="video/mp4" />
+</video>
+
+For a quick comparison, let's check out how the old and new approaches behave in typical scenarios.
+
+<video style="margin-bottom:24px" autoplay muted loop playsinline >
+  <source src="/static/blog/nextgen-pickers/quick-comparison-fields.mp4" type="video/mp4" />
+</video>
+
+Please, try it out for yourself in the live demo below:
+
+<iframe
+  src="https://codesandbox.io/embed/date-field-demo-pb87v0?fontsize=12&hidenavigation=1&module=%2F"
+  style="width:100%; height:200px; border:0; border-radius: 4px; overflow:hidden;"
+  allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+  sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+></iframe>
+
+### Fields roadmap
+
+The Fields are still in alpha, and there are many improvements on the way. The following list describes some of the most notable features planned.
+
+- **Smart pasting**
+
+  Date strings pasted in Fields will be parsed and matched with the sections of the date.
+
+- **Support days of the week**
+
+  Users will be able to select a day in the week just like the can select a month.
```

Comment:
Should we link to the github issues of those two ?

Reply:
I've added the links.